packages/core/src/RPA/core/windows/helpers.py

```diff
@@ -7,9 +7,12 @@
     import psutil
 
 
-def get_process_list() -> Dict:
+def get_process_list(get_process_info: bool = True) -> Dict:
     """Get process list.
 
     Returns dictionary mapping process id to process name
     """
```

Comment:
The parameter can be documented in the docstring.

Reply:
Yeah I can add a docstring.